engine/src/audio/SimpleSound.h

```diff
@@ -39,79 +41,84 @@
 
 namespace Audio {
 
-    // Forward definitions
-    class Stream;
-
-    /**
-     * Simple Sound abstract class
-     *
-     * @remarks This partial implementation implements foreground loading of files
-     *      using the codec registry.
-     *      @par No background loading is implemented, meaning all requests for load,
-     *      even with wait=false, are processed in the foreground.
-     *      @par There's a possibility for streaming given the packetized pulling
-     *      architecture. Renderers are not required to pull all packets from the stream,
-     *      and access to the Stream object is also provided for seeking back and forth.
-     *      @par Renderers still have to override (un)loadImpl() and abortLoad().
-     *      This refinement merely adds supporting methods for implementing them.
-     * @see Sound, BackgroundLoadingSound
-     *
+// Forward definitions
+class Stream;
+
+/**
+ * Simple Sound abstract class
+ *
+ * @remarks This partial implementation implements foreground loading of files
+ *      using the codec registry.
+ *      @par No background loading is implemented, meaning all requests for load,
+ *      even with wait=false, are processed in the foreground.
+ *      @par There's a possibility for streaming given the packetized pulling
+ *      architecture. Renderers are not required to pull all packets from the stream,
+ *      and access to the Stream object is also provided for seeking back and forth.
+ *      @par Renderers still have to override (un)loadImpl() and abortLoad().
+ *      This refinement merely adds supporting methods for implementing them.
+ * @see Sound, BackgroundLoadingSound
+ *
+ */
+class SimpleSound : public Sound {
+private:
```

Comment:
OK. I think the Google Style Guide says otherwise, but that's okay; we're already departing from that standard in several respects as it is. I will adjust this formatting rule and rerun it.

Reply:
Rerunning this would introduce a bunch more changes. I think I'll leave it for a later PR/discussion, actually